lib/userlike/test.js

```diff
@@ -72,4 +74,66 @@ describe('Userlike', function(){
       analytics.load(userlike, done);
     });
   });
+
+  describe('after loading', function(){
+
+    beforeEach(function(done){
+        analytics.once('ready', done);
+        analytics.initialize();
+        analytics.page();
+    });
+
+    describe('#listen', function(){
+
+      beforeEach(function(done){
+        analytics.once('ready', done);
+        analytics.initialize();
+        analytics.stub(analytics, 'track');
+      });
+
+      it('should send a chat started event', function(done){
+        setTimeout(function(){
+          userlikeStartChat();
+          done();
+        }, 3000);
+        setTimeout(function(){
+          analytics.called(analytics.track, 'Live Chat Conversation Started', {}, { context: { integration: { name: 'snapengage', version: '1.0.0' }}});
+          done();
+        }, 3000);
```

Comment:
FWIW, the order in which browsers handle timers *should* be consistent, but isn't always. If you're relying on the first callback having been executed before the second, you should nest your second setTimeout inside the first.

Also, `done()` should only be called once, and if it gets called before the assert on line 100, you may end up with a false positive test passage.

Gonna come chat with you real quick about this since I have a few other questions.